pulumi/python/requirements.txt

```diff
@@ -1,21 +1,21 @@
-awscli~=1.22.63
+awscli~=1.22.100
 grpcio==1.43.0
 fart~=0.1.5
 lolcat~=1.4
 nodeenv~=1.6.0
 passlib~=1.7.4
 pulumi-aws>=4.37.5
-pulumi-docker~=3.1.0
+pulumi-docker==3.2.0
 pulumi-eks>=0.37.1
-pulumi-kubernetes~=3.16.0
+pulumi-kubernetes==3.18.2
 pycryptodome~=3.14.0
 PyYAML~=5.4.1
 requests~=2.27.1
 setuptools==62.1.0
 setuptools-git-versioning==1.9.2
 wheel==0.37.1
 yamlreader==3.0.4
-pulumi-digitalocean==4.11.1
+pulumi-digitalocean==4.12.0
 pulumi-linode==3.7.1
 linode-cli~=5.17.2
-pulumi~=3.26.1
+pulumi~=3.30.0
```

Comment:
For increased consistency we might consider generating and shipping a `Pipfile.lock` to be sure the same versions are installed in various envs.  I'm not sure if we're using `pipenv` thought.  In any case just a small comment and likely out of scope for this PR.  Just curious to hear your thoughts.

Reply:
@dekobon does that in Bank of Sirius (or some variant of) - I'd really like to get that in place here, because there are definitely some versions we cannot upgrade to (currently pulumi-aws and py-yaml are at their highest version that still works). I'll open an issue.

Reply:
:+1: Pipfile.lock may be better than the way it is done in Bank of Sirius. I just copied what Google was doing in the existing project and added checksums.